test/unit/fh3/appforms/test_forms.js

```diff
@@ -51,5 +54,17 @@ module.exports = {
       assert.equal(err, null);
       return cb();
     });
+  },
+  'test appforms-forms export': function(cb) {
+    appformsforms.export({ file: 'test/fixtures/appforms/fixture_export.zip'}, function (err) {
+      assert.equal(err, null);
+      fs.unlink('test/fixtures/appforms/fixture_export.zip', cb);
```

Comment:
Maybe make this path the test zip to use with nock and instead write to os.tmpdir()